src/commands/deploy/deploy.ts

```diff
@@ -474,7 +477,14 @@ const runDeploy = async ({
     })
 
     config.headers = headers
-    uploadDeployBlobs({ deployId, siteId, silent, options, cachedConfig: command.netlify.cachedConfig })
+    await uploadDeployBlobs({
```

Comment:
How was this not causing problems without awaiting the Promise? 🤔

Reply:
honestly I don't know :D